rmgpy/molecule/group.py

```diff
@@ -472,22 +471,34 @@ def isSurfaceSite(self):
 
     def isOxygen(self):
         """
-        Return ``True`` if the atom represents an oxygen atom or ``False`` if
-        not.
+        Return ``True`` if the atom represents an oxygen atom or ``False`` if not.
         """
         allOxygens = [atomTypes['O']] + atomTypes['O'].specific
         checkList=[x in allOxygens for x in self.atomType]
-
         return all(checkList)
 
     def isSulfur(self):
         """
-        Return ``True`` if the atom represents an sulfur atom or ``False`` if
-        not.
+        Return ``True`` if the atom represents an sulfur atom or ``False`` if not.
         """
         allSulfur = [atomTypes['S']] + atomTypes['S'].specific
         checkList=[x in allSulfur for x in self.atomType]
+        return all(checkList)
 
+    def isNitrogen(self):
+        """
+        Return ``True`` if the atom represents an sulfur atom or ``False`` if not.
+        """
+        allNitrogen = [atomTypes['N']] + atomTypes['N'].specific
+        checkList=[x in allNitrogen for x in self.atomType]
+        return all(checkList)
+
+    def isCarbon(self):
```

Comment:
This is rather trivial, but you could consider placing these new methods in alphabetical order relative to `isOxygen` and `isSulfur`.